README.md

```diff
@@ -338,6 +338,14 @@ faster!
 
 ### *FixedSizeArrays* and *ImmutableArrays* compatibility
 
+Several existing packages for statically sized arrays have been developed for
+Julia, noteably *FixedSizeArrays* and *ImmutableArrays*. Upon consultation, it
+has been decided to move forward with *StaticArrays* which has found a new home
+in the *JuliaArrays* github organization. It is recommended that new users use
+this package, and that existing dependent packages consider switching to
+*StaticArrays* sometime during the life-cycle of Julia v0.5 as development for
```

Comment:
Nitpick - I'd simplify this to

"sometime during the life cycle of Julia v0.5."